### DISCUSSION TITLE
Possible Electrical Pathes

### DISCUSSION THREAD

**Discussion by Abbas111977** [SpiceSharp/SpiceSharp#188] [Q&A] [answered]
Hi there, 
Is it possible to use Spice# to find all possible electrical Kirchhoff paths for Diodes-Resistors-Capacitors network between 2 nods ?

![Example](https://user-images.githubusercontent.com/62871714/137449093-df69765e-d640-41f4-89f3-f1a1fd10af9b.jpg)

assuming that the nodes are the + and - of Vin, The output should be something like this: 

Path1: R1-R2
Path2: R1-R3
Path3: D2-R2
Path4: D2-R3

**Comment by svenboulanger** [comment, marked as answer]
This is outside the scope of Spice#. If you want to extract these loops, you will probably need to look up graph algorithms.

**Reply by Abbas111977** [reply]
Thank you Sir
